apps/api-journeys/src/lib/mockDb.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client'
+import { mockDeep, mockReset } from 'jest-mock-extended'
+import { DeepMockProxy } from 'jest-mock-extended/lib/cjs/Mock'
+
+import prisma from './db'
```

Comment:
this should match the exported name which is db

```suggestion
import db from './db'
```